instrumentation/opentelemetry-instrumentation-grpc/src/opentelemetry/instrumentation/grpc/__init__.py

```diff
@@ -145,10 +195,26 @@ class GrpcInstrumentorServer(BaseInstrumentor):
         grpc_server_instrumentor = GrpcInstrumentorServer()
         grpc_server_instrumentor.instrument()
 
+        If you want to add filters that only intercept requests
+        to match the condition, assign filters option to GrpcInstrumentorServer.
+
+        grpc_server_instrumentor = GrpcInstrumentorServer(
+            filters=filters.method_prefix("SimpleMethod"))
+        grpc_server_instrumentor.instrument()
+
     """
 
     # pylint:disable=attribute-defined-outside-init, redefined-outer-name
 
+    def __init__(self, filters=None):
```

Comment:
Should `filters` be singular -> `filter`? Also, please add type annotation for the param

Reply:
Though I wanted to use singular, `filter` is the name of builtin function and that's the reason why I use plural here. I would like to choose better naming if possible.

Reply:
Ah good point, I think one option would be `filter_`. I'll leave it up to you tho

Reply:
yeah, I used `filter_` first but I refrained from using underscore as I thought it's not recommended in this repo. If it's okay, I'm happy to replace the variable name as you say.

Reply:
yeah, I used `filter_` first but I refrained from using underscore as I thought it's not recommended in this repo. If it's okay, I'm happy to replace the variable name as you say.

Reply:
yeah, I used `filter_` first but I refrained from using underscore as I thought it's not recommended in this repo. If it's okay, I'm happy to replace the variable name as you say.

Reply:
@aabmass I talked with my friend about this and I heard from him that SQLAlchemy is using underscore to avoid the conflict with both keywords and builtin.
https://github.com/sqlalchemy/sqlalchemy/blob/6a496a5f40efe6d58b09eeca9320829789ceaa54/lib/sqlalchemy/ext/serializer.py#L81

I would like to leave the decision to the approvers and maintainers, but I hope to have make this discussion to be the convention in this project, and the rules for it in the linter. This allows upcoming contributors can apply the guideline without confusion.

@srikanthccv WDYT?

Reply:
FYI: flake8 seems to have a plugin to achieve it. 
https://pypi.org/project/flake8-builtins/

Reply:
Can you prefix it with something like `service_filter` or similar? I am fine with underscore too.

Reply:
@srikanthccv thank you for the comment. Because this may include more filters that should cover other than services, I'm leaning towards `filter_` at this momoent.